packages/web-react/src/components/Box/demo/BoxWithBorder.tsx

```diff
@@ -19,10 +16,7 @@ const BoxWithBorder = () => (
     <Box padding="space-800" borderWidth="200" borderRadius="300">
       With thicker border
     </Box>
-    <Box padding="space-800" borderWidth="100" borderRadius="300" borderColor="focus">
-      With focus color
-    </Box>
-    <Box padding="space-800" borderWidth="100" borderStyle={BorderStyles.DASHED}>
+    <Box padding="space-800" borderWidth="100" borderStyle="dashed">
```

Comment:
why not const? 
```suggestion
    <Box padding="space-800" borderWidth="100" borderStyle={BorderStyles.DASHED}>
```

Reply:
And why just here and not in other props?